graylog-storage-elasticsearch6/src/main/java/org/graylog/storage/elasticsearch6/IndicesAdapterES6.java

```diff
@@ -224,12 +227,20 @@ public void create(String indexName, IndexSettings indexSettings, String templat
 
     @Override
     public boolean ensureIndexTemplate(String templateName, Map<String, Object> template) {
-        final PutTemplate request = new PutTemplate.Builder(templateName, template).build();
+        final PutTemplate request = new PutTemplate.Builder(uncheckedURLEncode(templateName), template).build();
```

Comment:
@dennisoelkers This is only needed for the ES 6 adapter because the ES 7 client is automatically escaping it?

Reply:
@bernd: Correct. For ES7 I did not notice issues during index creation/rotation/deflector cycle beyond the issues in the `MongoIndexSet` class.

Reply:
@dennisoelkers Alright, thanks for the clarification!  :+1: